paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/lookup/FullCacheLookupTable.java

```diff
@@ -87,6 +84,7 @@ public abstract class FullCacheLookupTable implements LookupTable {
     private Predicate specificPartition;
 
     public FullCacheLookupTable(Context context) {
+        super(context.table);
```

Comment:
Some fields of this class  such as `refreshExecutor`, `maxPendingSnapshotCount`.. are extracted to the parent class, so can be removed now.